frontend/packages/kubevirt-plugin/src/components/vm-like/vm-like-resource-link.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import * as classNames from 'classnames';
+import { connectToModel } from '@console/internal/kinds';
+import { ResourceIcon, resourcePath } from '@console/internal/components/utils';
+import { getName, getNamespace } from '@console/shared';
+import { VirtualMachineInstanceModel, VirtualMachineModel } from '../../models';
+import { isVM } from '../../selectors/vm';
+
+export const VMLikeResourceLink = connectToModel(({ className, inline = false, title, obj }) => {
+  const name = getName(obj);
+  const namespace = getNamespace(obj);
+
+  const VMpath = resourcePath(VirtualMachineModel.kind, name, namespace);
```

Comment:
shouldn't this kind be conditional as well?

Reply:
The idea of this component is to always point to the `.../virtualmachines/...` path.
so if someone press the VMI link we want them to always end up in the  `.../virtualmachines/...` page.

Reply:
looked at the docs, no need to share paths, yeee :-) , removed the code that share vm and vmi paths :+1: